elements/pl-symbolic-input/pl-symbolic-input.mustache

```diff
@@ -85,3 +103,11 @@ Your answer must be a symbolic expression. All numbers must be rational - so, <c
 {{#shortformat}}
 symbolic expression
 {{/shortformat}}
+
+{{#format_error}}
+    <hr>
+    <p> Invalid format: the submitted answer could not be interpreted as a symbolic expression.  Expressions can be a mixture of explicit operators and allowable constants.  Function arguments must be surrounded by a pair of parentheses <code class="user-output">()</code>.  See the input hint text for more relevant information.
```

Comment:
Can we include the full details in this popover? I'm worried that students won't know what the "input hint" refers to. I like the way this works with `pl-number-input`, where there is no reference to the input hint and we just give all the information in the error popover.

Reply:
Changed to reference the format hint:

<img width="482" alt="image" src="https://user-images.githubusercontent.com/1790491/75057511-cbdc4d80-549e-11ea-82ff-b7e705b40769.png">